source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -106,14 +106,15 @@
 
 /*---------------------------------------------------------------------------*/
 
-/*
- * pdFREERTOS_ERRNO_EAFNOSUPPORT
```

Comment:
Something is strange here: this diff shows that FreeRTOS_IP.c has changed: a comment about `pdFREERTOS_ERRNO_EAFNOSUPPORT` was removed.

But when I compare [FreeRTOS:main:FreeRTOS_IP.c](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/source/FreeRTOS_IP.c) with [holden-zenith:dev-network-interface/FreeRTOS_IP.c](https://github.com/holden-zenith/FreeRTOS-Plus-TCP/blob/dev-network-interface/source/FreeRTOS_IP.c), I see zero differences.

Reply:
that comment was in the config defaults. I had originally intended for it to be used to check if the kernel was up to date but I don't think it ever got used anyway.